jujugui/static/gui/src/app/components/search-results-item/search-results-item.js

```diff
@@ -194,6 +194,44 @@ YUI.add('search-results-item', function(Y) {
       });
     },
 
+    /**
+      Generate the series list item class based on entity type
+
+      @method _generateSeriesClass
+      @returns {String} The generated class name.
+    */
+    _generateSeriesClass: function() {
+      var item = this.props.item;
```

Comment:
This could be `var type = this.props.item.type` to save you referencing the type attribute multiple times below.